src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -2492,18 +2492,13 @@ void Interpolate::prepareParams() {
 
     auto buildExecutor = [&](const InterpolateKey& key) -> std::shared_ptr<InterpolateExecutorBase> {
         std::shared_ptr<InterpolateExecutorBase> executor;
-        if ((key.nodeAttrs.mode == InterpolateMode::nearest || key.nodeAttrs.mode == InterpolateMode::linear_onnx ||
-             key.nodeAttrs.mode == InterpolateMode::cubic) &&
-            ((key.nodeAttrs.layout != InterpolateLayoutType::planar && mayiuse(cpu::x64::sse41)) ||
-             (mayiuse(cpu::x64::avx2) && key.nodeAttrs.inPrc == ov::element::f32))) {
-            executor = std::make_shared<InterpolateJitExecutor>(key.nodeAttrs,
-                                                                key.srcDims,
-                                                                key.dstDims,
-                                                                key.dataScales,
-                                                                key.attr);
-        } else if ((key.nodeAttrs.mode == InterpolateMode::bilinear_pillow ||
-                    key.nodeAttrs.mode == InterpolateMode::bicubic_pillow) &&
-                   (key.nodeAttrs.layout == InterpolateLayoutType::by_channel)) {
+        if (((key.nodeAttrs.mode == InterpolateMode::nearest || key.nodeAttrs.mode == InterpolateMode::linear_onnx ||
```

Comment:
bool jit_nearest_liniear_cubic
bool jit_pillow

Reply:
Split condition parts into several variables this if statements